### DISCUSSION TITLE
Can I use ConvertToPolar for more than two traits?

### DISCUSSION THREAD

**Discussion by ferfacal92** [UMCUGenetics/PolarMorphism#7] [Q&A] [answered]
Hello,

I wonder if I can use ConvertToPolar for more than two traits? In my case for 3 traits.

Thank you

Fernando

**Comment by Joannavonberg** [comment, marked as answer]
Hi @ferfacal92, thank you for your question. Yes, ConvertToPolar can definitely be used for more than two traits. I looked at the function and its documentation, and realized I didn't update the documentation after I figured out how to do the whole analysis for more than two traits. I made a new issue ( #(8) ) for this, I will start working on it. You can subscribe to this issue to get an update when I have changed it. In the mean time you can already try to use the function, since I will not change anything in the code (just the documentation). You can use this 'tutorial' I wrote as a guide: https://github.com/UMCUGenetics/PolarMorphism/blob/master/template_for_github.Rmd (specifically, line 48: `CES.AF <- ConvertToPolar(dfnames = traits, snpid = "snpid", whiten = T, LDcorrect = F)`)

**Comment by ferfacal92** [comment]
Thank you for your answer

Kind regads
Fernando Facal
________________________________
De: Joanna ***@***.***>
Enviado: viernes, 27 de octubre de 2023 16:49
Para: UMCUGenetics/PolarMorphism ***@***.***>
Cc: ferfacal92 ***@***.***>; Mention ***@***.***>
Asunto: Re: [UMCUGenetics/PolarMorphism] Can I use ConvertToPolar for more than two traits? (Discussion #7)


Hi @ferfacal92<https://github.com/ferfacal92>, thank you for your question. Yes, ConvertToPolar can definitely be used for more than two traits. I looked at the function and its documentation, and realized I didn't update the documentation after I figured out how to do the whole analysis for more than two traits. I made a new issue ( #(8) ) for this, I will start working on it. You can subscribe to this issue to get an update when I have changed it. In the mean time you can already try to use the function, since I will not change anything in the code (just the documentation). You can use this 'tutorial' I wrote as a guide: https://github.com/UMCUGenetics/PolarMorphism/blob/master/template_for_github.Rmd (specifically, line 48: CES.AF <- ConvertToPolar(dfnames = traits, snpid = "snpid", whiten = T, LDcorrect = F))
